src/MessageTile/MessageTileButton.jsx

```diff
@@ -26,7 +26,20 @@ MessageTileButton.propTypes = {
   /**
    * The onClick event for the Button
    */
-  onClick: PropTypes.func
+  onClick: PropTypes.func,
+  /**
+   * If the button should run full width
+   */
+  fullWidth: PropTypes.bool,
+  /**
+   * If the button should be a little wider, having extra padding left and right
+   */
+  prominent: PropTypes.bool
```

Comment:
`prominent` really makes it just a tiny bit wider. Had a hard time finding the difference when toggling this prop.

- With `prominent` `false`

![image](https://user-images.githubusercontent.com/1538066/64870130-18328000-d619-11e9-8717-64132dd9e74a.png)

- With `prominent` `true`

![image](https://user-images.githubusercontent.com/1538066/64870142-1ec0f780-d619-11e9-94b5-9492f36f1623.png)

Reply:
Good point @tiagoamaro! It's unclear from a UI perspective why this is a prop.

Reply:
It's a prop because the underlying component (the `Button`) accepts `prominent` as a prop. This makes the MessageTile be able to 100% replicate the design of the [ArticleDonationBanner](http://styleguide.theconversation.com/?path=/story/promos--articledonationbanner). My goal is to deprecate the ArticleDonationBanner, and instead use MessageTile.

If you're amenable to it, we could look at removing the distinction in `Button` (which is our code), and then not having it as a concept at all; the button on the end of article call-to-action would change ever so slightly (the button would be slightly less wide), but everything would be more uniform and consistent.

Reply:
If button accepts that prop already, can't a regular Button just be a child within MessageTile?

Reply:
The `MessageTileButton` builds the button rather than accepts it as a child, attaching the `onClick` and adding styles (adding them via the theming).

A regular `Button` _can definitely_ be a child within `MessageTile`, but it won't pick up the margin.

Reply:
If you have a better way to structure it, I'm all ears. 

Also, my opinion is to just remove `prominent` from `Button`, and reduce the number of options. Then, fewer differing types of buttons, and no need to include it as a configurable prop here.

I think it was added in 86caab8e38fbf88db766b86541358f35a1a59cf1.

Reply:
> If button accepts that prop already, can't a regular Button just be a child within MessageTile? - @nickbrowne 

> The `MessageTileButton` builds the button rather than accepts it as a child... - @markcipolla 

### Suggestion

Since the `MessageTileButton` is a special MUI `Button` component, with custom `marginBottom` spacing, it could simply accept all props of a regular `Button`, so it would as flexible as possible whenever rendering them:

```jsx
export const MessageTileButton = ({ children, classes, ...props }) => {
  return (
    <Button {...props} className={classes.button} colour='primary'>
      {children}
    </Button>
  )
}

MessageTileButton.defaultProps = {
  fullWidth: true,
  prominent: false
}
```

Same behaviour, also having default props documented, but allowing all regular MUI `Button` props to be passed to it. Just would need to document that `className` and `colour` are constant values.

### Screenshot

![image](https://user-images.githubusercontent.com/1538066/64964069-b4a28f80-d870-11e9-9660-3497287222e8.png)

Reply:
@nickbrowne thoughts? I think that is the best of both worlds. 👏 Good idea @tiagoamaro !

> ...but allowing all regular MUI Button props to be passed to it. Just would need to document that className and colour are constant values.

Very nice. `MessageTileButton` goes back to knowing less about the implementation of `Button`, as it should.

Also, after chatting with @ZoeJazz and @jameshill, we have made a item in the [TCUI transition checklist](https://trello.com/c/xkAe8637/747-tcui-transition) to look at removing `prominent` from `Button`, and the components which also use it.